server/libs/infra/src/search/schemas/asset.schema.ts

```diff
@@ -0,0 +1,37 @@
+import { CollectionCreateSchema } from 'typesense/lib/Typesense/Collections';
+
+export const assetSchemaVersion = 1;
+export const assetSchema: CollectionCreateSchema = {
+  name: `assets-v${assetSchemaVersion}`,
+  fields: [
+    // asset
+    { name: 'ownerId', type: 'string', facet: false },
```

Comment:
The schema is referenced from the entity JSON, correct?

Reply:
Yes, the field names are using dot notation to pull out the relevant data from the "document", which in this case is the `AssetEntity` with all of it's relations loaded.